tests/integration/suite/test_cluster_defaults.py

```diff
@@ -107,6 +107,17 @@ def test_import_initial_conditions(admin_mc, remove_resource):
     assert cluster.conditions is None
 
 
+def test_cluster_initial_kubeconfig(admin_mc, remove_resource):
```

Comment:
This isn't going to test the fix. A default "empty" cluster gets a zero value for localClusterAuthEndpoint so it passes even without the fix.

Reply:
I cherry picked to master and it does test the fix. The value being omitted on create has the same effect and leads to the zero value of nil which causes the panic.